src/react-apps/applications/shared/src/components/AltinnSearchInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormControl, InputAdornment, TextField, createMuiTheme } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import * as React from 'react';
+import altinnTheme from '../theme/altinnStudioTheme';
+
+export interface IAltinnSearchInputCompontentProvidedProps {
+  classes: any;
+  id: string;
+  placeholder: any;
+  onChangeFunction: any;
+}
+
+export interface IAltinnSearchInputComponentProps extends IAltinnSearchInputCompontentProvidedProps {
```

Comment:
Unecessary interface, since it doesn't expand anything.

Reply:
removed